docs/docs/CMDB/FAQ.md

```diff
@@ -52,6 +52,35 @@ device.lab(config-router-bgp-af)#neighbor 100.64.1.0 ?
   weight Assign weight for routes learnt from this peer
 ```
 
+## Why is there no permit/deny field in community-lists and prefix-lists?
+
+While some devices provides a `permit/deny` field to a community/prefix-list, other Network OS like JunOS does not. More importantly, OpenConfig does not have it either.
+
+If you are currently using `permit/deny`, we suggest you to adapt your route-map instead.
+
+!!! example
+
+    You should migrate from:
+    ```
+    ip prefix-list FANCY_PREFIX_LIST 10 deny 10.0.0.0/24
```

Comment:
nitpick: the example is not reflecting a real use-case, it makes sense to have multiple sequence with a combination of deny/permit if the prefixes overlap.

I would propose something like this:
```
ip prefix-list FANCY_PREFIX_LIST 10 deny 10.0.0.0/24
ip prefix-list FANCY_PREFIX_LIST 20 permit 10.0.0.0/8 le 32
```
:arrow_up: here we want all prefixes within 10/8 except 10.0.0.0/24

Reply:
done